components/Buttons/src/MDCFloatingButton.m

```diff
@@ -19,9 +19,49 @@
 #import "MaterialShadowElevations.h"
 #import "private/MDCButton+Subclassing.h"
 
+#import <MDFInternationalization/MDFInternationalization.h>
+
 static const CGFloat MDCFloatingButtonDefaultDimension = 56.0f;
 static const CGFloat MDCFloatingButtonMiniDimension = 40.0f;
+static const CGFloat MDCFloatingButtonDefaultImageTitleSpacing = 8;
+static const UIEdgeInsets internalLayoutSpacingInsets = (UIEdgeInsets){0, 16, 0, 24};
+
 static NSString *const MDCFloatingButtonShapeKey = @"MDCFloatingButtonShapeKey";
+static NSString *const MDCFloatingButtonModeKey = @"MDCFloatingButtonModeKey";
+static NSString *const MDCFloatingButtonImageLocationKey = @"MDCFloatingButtonImageLocationKey";
+static NSString *const MDCFloatingButtonImageTitleSpacingKey = @"MDCFloatingButtonImageTitleSpacingKey";
+
+static UIEdgeInsets UIEdgeInsetsFlippedHorizonally(UIEdgeInsets insets) {
+  return UIEdgeInsetsMake(insets.top, insets.right, insets.bottom, insets.left);
+}
+
+// TODO(rsmoore): All properties within here will be made public in a future PR.
```

Comment:
Super-nit: Consider opening an issue and making this "TODO(#3218): All properties within here will be made public in a future PR."

Reply:
I figured since that would be the next PR on this feature branch, it was OK to just remark for my reviewers. I can open an issue if you like.